Tasks/AzureKeyVaultV1/operations/azure-arm-keyvault.ts

```diff
@@ -59,7 +59,8 @@ export class KeyVaultClient extends azureServiceClient.ServiceClient {
 
                 return response;
             } catch(error) {
-                if (retriableErrorCodes.indexOf(error.code) != -1 && ++retryCount < maxRetryCount) {
+                if ((retriableErrorCodes.indexOf(error.code) != -1 || error.toString().indexOf('Request timeout: ') != -1)
```

Comment:
It is not a big thing but I wonder if it works with `startsWith`? 
`error.toString().indexOf('Request timeout: ') != -1`
to
`startsWith('Request timeout: ')`

Reply:
Haven't checked but it might not work if toString() doesn't put message at the beginning of serialized string

Reply:
I have recommended it because if it is always in the beginning of string, as far as I know `startsWith` will work faster since it will fail fast without searching whole string.

Reply:
so `startsWith` is not a solution here?